src/main/kotlin/ir/jaamebaade/jaamebaade/service/AuthenticationService.kt

```diff
@@ -28,4 +39,14 @@ class AuthenticationService(
         return user.id
 
     }
+    fun login() : UserDetails? {
```

Comment:
why do we need to implement this function?

Reply:
Login: for login with token
LoginByCredentials: for login by username and password?